intro-HPC/antwerpen/ch_monitoring_utilization.tex

```diff
@@ -3,24 +3,25 @@ \chapter{Monitoring Cluster Utilisation}
 
 \section{Introduction}
 
-As the UA-HPC is rapidly growing, the need to efficiently monitor the available
+As the \hpc is rapidly growing, the need to efficiently monitor the available
 computer resources is more important than ever. The term \emph{monitor} when
-applied to the UA-HPC centre can be confusing since it means different things
+applied to the \hpc center can be confusing since it means different things
 depending on who is saying it and who is hearing it.
 
 For example:
 
 \begin{enumerate}
 \item  The \strong{end-users} (persons running applications on the cluster) think: ``When will my job start and when will it be finished? Did I specify the right computer resources?  How is it performing compared to last time and can I speed it up?''
-\item  The \strong{systems engineers} of the UA-HPC think: ``How are our machines performing? Are all the services functioning correctly? What trends do we see and how can we better utilise our overall computer resources?''
+\item  The \strong{systems engineers} of the \hpc think: ``How are our machines
+performing? Are all the services functioning correctly? What trends do we see and how can we better utilize our overall computer resources?''
 \end{enumerate}
 
-The UA-HPC provides an UA-HPC utilisation information system, which is based on Ganglia.
+The \hpc provides an \hpc utilization information system, which is based on Ganglia.
 
 ``\emph{Ganglia}'' is a system-monitoring tool for high-performance computing
 systems, letting users view live or historical statistics (such as CPU load
-averages or network utilisation) in a graphical manner. Ganglia are monitoring
-all the machines in the UA-HPC.
+averages or network utilization) in a graphical manner. Ganglia are monitoring
```

Comment:
I’ve also changed this. It is of course confusing that the file names still use the z.

And in fact, Oxford Dictionary prefers utilize over utilise. This is what I get when I search for utilise: http://www.oxforddictionaries.com/definition/english/utilize?q=utilise&searchDictCode=all

It’s hard to be consistent in the s/z-issues these days. Even the British are more and more switching over to the z.

Regards,

Kurt


Van: Jens Timmerman <notifications@github.com<mailto:notifications@github.com>>
Beantwoorden - Aan: hpcugent/vsc_user_docs <reply@reply.github.com<mailto:reply@reply.github.com>>
Datum: donderdag 5 februari 2015 11:53
Aan: hpcugent/vsc_user_docs <vsc_user_docs@noreply.github.com<mailto:vsc_user_docs@noreply.github.com>>
CC: Kurt Lust <Kurt.Lust@uantwerpen.be<mailto:Kurt.Lust@uantwerpen.be>>
Onderwerp: Re: [vsc_user_docs] Change of names for the University of Antwerp. (#94)


In intro-HPC/antwerpen/ch_monitoring_utilization.tex<https://github.com/hpcugent/vsc_user_docs/pull/94#discussion_r24155860>:

>
>  ``\emph{Ganglia}'' is a system-monitoring tool for high-performance computing
>  systems, letting users view live or historical statistics (such as CPU load
> -averages or network utilisation) in a graphical manner. Ganglia are monitoring
> -all the machines in the UA-HPC.
> +averages or network utilization) in a graphical manner. Ganglia are monitoring


I believed we had agreed on the British spelling?

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/vsc_user_docs/pull/94/files#r24155860>.